### ISSUE TITLE
[Docs] Extend docs for custom builds

### ISSUE THREAD

**Issue by ma2ciek** [ckeditor/ckeditor5-angular#71]
I've just hit the https://github.com/ckeditor/ckeditor5-angular/issues/20 issue. Locally saved `ckeditor.js` as CKEditor 5 build doesn't work when targetting `ES5`. I needed to set `target: 'ES6'` in `tsconfig.json`.

There's also a related problem that users need to know when using custom builds:
`allowJs: true` has to be used when `strict` option is set to `true`. Otherwise the following problem occurs:

```
Could not find a declaration file for module '../../ckeditor.js'. '/Users/maciejbukowski/workspace/ckeditor5-angular/src/ckeditor.js' implicitly has an 'any' type
```

**Comment by Routhinator** [comment]
I've hit issue #20 now and was looking at getting help on StackOverflow https://stackoverflow.com/questions/56611296/how-to-properly-build-angular-8-and-ckeditor5-angular-for-prod-mode but it went away when I disabled optimization in the prod build. 

It happens with both a custom build thats published on npm or even the vanilla classic build. I don't have strict mode enabled, and I added `allowJs: true` just to see if it would fix it, and it didn't. So there is yet another related issue. But as it stands I cannot get the latest version of ckeditor5-angular when in an optimized build at all following the docs, and I've tried Angular 6, 7 and 8.
